src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -32,15 +46,13 @@ const listStyles = ({ theme }) => css`
   `};
 `;
 
-const ButtonGroupList = styled('ul')`
-  ${listStyles};
-`;
+const ButtonGroupList = styled('ul')(baseStyles, alignmentStyles);
 
 /**
  * Groups its Button children into a list and adds margins between.
  */
-const ButtonGroup = ({ children }) => (
-  <ButtonGroupList>
+const ButtonGroup = ({ children, align }) => (
```

Comment:
Can you put the constants on the component? For example `ButtonGroup.LEFT = LEFT;`. The user can then be sure to use the right values directly.